php/php-zstd/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           php 1.1
+
+github.setup        kjdev php-ext-zstd 0.7.4
+fetch.type          git
```

Comment:
is setting the `fetch.type` necessary? That's unusual to set while using the GitHub portgroup.

Reply:
Yeah, I think (from memory), that it's needed because the repo makes use of git modules and , without the fetch.type set to git... git modules cannot be installed, ultimately failing.

I've just tried here without the fetch type and I get:

````
notice:fetch --->  Fetching distfiles for php73-zstd
:debug:fetch Executing org.macports.fetch (php73-zstd)
:debug:fetch Executing proc-post-org.macports.fetch-fetch-0
:debug:fetch system -W /opt/local/var/macports/build/_opt_local_localports_php_php-zstd/php73-zstd/work/php-ext-zstd-0.7.4: git submodule update --init
:info:fetch chdir: /opt/local/var/macports/build/_opt_local_localports_php_php-zstd/php73-zstd/work/php-ext-zstd-0.7.4: No such file or directory
:info:fetch Command failed: git submodule update --init
:info:fetch Exit code: 1
:error:fetch Failed to fetch php73-zstd: command execution failed
````

Reply:
It really would be better if the extension, instead of git modules to incorporate stuff was using another way to get zstandard-libs/headers at hand, but it's the way it makes it, I'm afraid.